tests/federation_rooms_invite_test.go

```diff
@@ -17,85 +18,80 @@ func TestFederationRoomsInvite(t *testing.T) {
 	alice := deployment.Client(t, "hs1", "@alice:hs1")
 	bob := deployment.Client(t, "hs2", "@bob:hs2")
 
-	t.Run("Parallel", func(t *testing.T) {
-		// sytest: Invited user can reject invite over federation
-		t.Run("Invited user can reject invite over federation", func(t *testing.T) {
-			t.Parallel()
-			roomID := alice.CreateRoom(t, map[string]interface{}{
-				"preset": "private_chat",
-			})
-			alice.InviteRoom(t, roomID, bob.UserID)
-			bob.MustSyncUntil(t, client.SyncReq{}, client.SyncInvitedTo(bob.UserID, roomID))
-			bob.LeaveRoom(t, roomID)
-			alice.MustSyncUntil(t, client.SyncReq{}, client.SyncLeftFrom(bob.UserID, roomID))
-		})
+	roomID := alice.CreateRoom(t, map[string]interface{}{
+		"preset": "private_chat",
+		"name":   "Invites room",
+	})
 
-		// sytest: Invited user can reject invite over federation several times
-		t.Run("Invited user can reject invite over federation several times", func(t *testing.T) {
-			t.Parallel()
-			roomID := alice.CreateRoom(t, map[string]interface{}{
-				"preset": "private_chat",
-			})
-			for i := 0; i < 3; i++ {
-				alice.InviteRoom(t, roomID, bob.UserID)
-				bob.MustSyncUntil(t, client.SyncReq{}, client.SyncInvitedTo(bob.UserID, roomID))
-				bob.LeaveRoom(t, roomID)
-				alice.MustSyncUntil(t, client.SyncReq{}, client.SyncLeftFrom(bob.UserID, roomID))
-			}
-		})
+	aliceSince := ""
+	bobSince := ""
+
+	// sytest: Invited user can reject invite over federation
+	t.Run("Invited user can reject invite over federation", func(t *testing.T) {
+		alice.InviteRoom(t, roomID, bob.UserID)
+		bobSince = bob.MustSyncUntil(t, client.SyncReq{Since: bobSince}, client.SyncInvitedTo(bob.UserID, roomID))
+		bob.LeaveRoom(t, roomID)
+		aliceSince = alice.MustSyncUntil(t, client.SyncReq{Since: aliceSince}, client.SyncLeftFrom(bob.UserID, roomID))
+	})
 
-		// sytest: Invited user can reject invite over federation for empty room
-		t.Run("Invited user can reject invite over federation for empty room", func(t *testing.T) {
-			t.Parallel()
-			roomID := alice.CreateRoom(t, map[string]interface{}{
-				"preset": "private_chat",
-			})
-			aliceSince := alice.MustSyncUntil(t, client.SyncReq{}, client.SyncJoinedTo(alice.UserID, roomID))
+	// sytest: Invited user can reject invite over federation several times
+	t.Run("Invited user can reject invite over federation several times", func(t *testing.T) {
+		for i := 0; i < 3; i++ {
 			alice.InviteRoom(t, roomID, bob.UserID)
-			charlieSince := bob.MustSyncUntil(t, client.SyncReq{}, client.SyncInvitedTo(bob.UserID, roomID))
```

Comment:
not sure what `charlieSince` was here.  A typo?

Reply:
More or less, probably had a Charlie before and forgot to rename the variable. :)